projects/igniteui-angular/src/lib/dialog/dialog.component.ts

```diff
@@ -83,6 +83,7 @@ export class IgxDialogComponent implements IToggleView, OnInit, OnDestroy, After
         this._isModal = val;
     }
 
+    @Input()
```

Comment:
Umm what is this? Where the API description comment? What's the default? Mark as public as well.